zeppelin/cli.py

```diff
@@ -19,9 +23,23 @@ def main():
     else:
         args.out_filename = 'knowledge'
 
-    zeppelin_converter = ZeppelinConverter(args.in_filename, args.out_filename,
-                                           directory, 'anonymous', 'N/A', 'N/A')
-    zeppelin_converter.convert()
+    try:
+        with open(args.in_filename, 'rb') as raw:
+            t = json.load(raw)
+            full_path = os.path.join(directory, args.out_filename + '.md')
+            with open(full_path, 'w') as fout:
+                if 'results' in t['paragraphs'][0]:
```

Comment:
Can you put this logic in a function please? Even if it's just a one line, I think it's worth it for example for testing